changesets2CSV/changesets2CSV

```diff
@@ -222,26 +622,34 @@ def changeset_csv(output_file, changesets, name=None, summary=False):
 
                 csv_writer.writerow(changeset)
 
+
     except Exception as e:
         print("Error creating csv {}".format(e))
         raise
 
     if summary:
-        return {'Changesets': changeset_count,
+        summary_dict = {'Changesets': changeset_count,
                 'Edits': edit_count,
                 'Additions': add_count,
                 'Modifications': modify_count,
                 'Deletions': delete_count,
                 'Discussions': discussion_count,
                 'Edits/Changeset': round((edit_count/changeset_count), 2)}
 
+        #Include tag-change counts into the summary dictionary
+        for k,v in tag_change_totals.items():
```

Comment:
You should be able to extend the dict: `summary_dict.update(tag_change_totals)` instead of the loop